alphagov/paas-cf
# [#153119879] Remove bosh-lite tooling

## What

Remove bosh-lite tooling.

We don't think that anyone on the team uses this anymore  so it hasn't been
well maintained. So we don't think it's worth the effort of updating and
testing it against a new version of Terraform.

With apologies to Hector!

## How to review

1. Check my rationale makes sense.
1. Check that I've removed all of the things related to bosh-lite.'

## Who can review

Not @46bit or myself.
